app/assets/stylesheets/forms.scss

```diff
@@ -1,3 +1,14 @@
 .govuk-label {
   font-weight: bold;
 }
+
+div#votes p  {
+display: inline;
+float: right;
```

Comment:
It's a good idea in general to [avoid using float](https://www.quora.com/Should-the-float-attribute-be-avoided-in-modern-CSS-and-why) unless you need it.

What you're doing here can be done with `display: inline-block;` and `text-align: right`